src/lib/kit/components/SimpleVerticalAccordeon/SimpleVerticalAccordeon.tsx

```diff
@@ -97,19 +101,38 @@ export class SimpleVerticalAccordeon extends React.Component<
             return null;
         }
 
+        const title = this.getTitle();
+        const titlePopoverDisabled =
+            (this.titleRef.current?.offsetWidth || 0) < MAX_TITLE_TEXT_WIDTH;
```

Comment:
`const titlePopoverDisabled =
            (this.titleRef.current?.offsetWidth || 0) <= MAX_TITLE_TEXT_WIDTH;`